tools/champ_blocs/div_index.xml

```diff
@@ -0,0 +1,84 @@
+<tool id="cb_div" name="Diversity" version="@VERSION@" profile = "20.01">
+    <description>indicators</description>
+    <macros>
+        <import>macro.xml</import>
+    </macros>
+    <expand macro="cb_requirements">
+        <requirement type="package" version="3.3.6">r-ggplot2</requirement>
+        <requirement type="package" version="1.7_20">r-ade4</requirement>
+        <requirement type="package" version="2.2">r-adiv</requirement>
+        <requirement type="package" version="2.0.3">r-magrittr</requirement>
+    </expand>
+    <required_files>
+        <include type="literal" path="test-data/Clean_Rdata.Rdata"/>
+        <include type="literal" path="div_index.r"/>
+    </required_files>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript
+            '$__tool_directory__/div_index.r'
+            '$input_data'
+            '$output_div'
+            '$output_rdata'
+            '$plots'
+        ]]>
+    </command>
+    <inputs>
+        <param name="input_data" type="data" format="data" label="Input Clean data"/>
```

Comment:
the type is is very generic, anything better that we can use here?

Reply:
I tried to put the format as Rdata but it does not detect the output 'Clean Rdata' from the cb_dissimlarity tool.

Reply:
Try `rdata` the real datatype is called that way see here: https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/config/sample/datatypes_conf.xml.sample#L670

Please chance it everywhere, I hope it will work then